tests/acceptance/test_build.py

```diff
@@ -948,6 +948,9 @@ def test_build_artifact_depends_and_provides(
         # Then remove it, not to mess up the expected test output
         del other.provides["rootfs_image_checksum"]
 
+        # MEN-3076: Mender-Artifacts writes software version by default
+        dependsprovides.provides.update({'rootfs-image.version': 'mender-image-master'})
```

Comment:
Meta-mender is used by several different Mender versions, this will break with older versions. I propose to delete the key instead, as we do for `rootfs_image_checksum`. This is tested in so many other places that I don't think it's critical to test it here.

Reply:
Ack, I wasn't aware of this back-dependency on older versions, I've updated the test.